### PR TITLE
Js2 3

### PR DESCRIPTION
**Pull request by OleksiiTarasenko**
js 2 & 3

### CODE REVIEW

File: ba/js/lesson2/script.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    break;
+  case "8":
+    console.log("*");
+    break;
+  case "9":
+    console.log(")");
+    break;
+  default:
+    console.log("Wrong button!!!");
+    break;
```
Comment (idovbush) [inline ba/js/lesson2/script.js]: we do not need break in default section

File: ba/js/lesson2/script.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+switch (currency) {
+  case "EUR":
+    console.log(`You have got ${sum * EX_RATE_EUR} euros!`);
+    break;
+  case "UAH":
+    console.log(`You have got ${sum * EX_RATE_UAH} hryvnyas!`);
+    break;
+  case "AZN":
+    console.log(`You have got ${sum * EX_RATE_AZN} manats!`);
+    break;
```
Comment (idovbush) [inline ba/js/lesson2/script.js:102]: 'default' should almost always be mandatory

File: ba/js/lesson3/script.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
+//Task 1
+
+let start = prompt(" Enter start number");
+let end = prompt("Enter end number");
+let sum = 0;
+for (let i = +start; i < +end + 1; i++) {
```
Comment (idovbush) [inline ba/js/lesson3/script.js]: it better not to write it like that let start = +prompt(" Enter start number") or let start = Number(prompt(" Enter start number"));

File: ba/js/lesson3/script.js [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+  sum += i;
+}
+
+alert(`Total: ${sum}`);
+//Task 2
+let numA = prompt(" Enter number A");
```
Comment (idovbush) [inline ba/js/lesson3/script.js:12]: the same Number( prompt(" Enter number A");)

File: ba/js/lesson3/script.js [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+let b = +numB;
+
+while (a != 0 && b != 0) {
+  a > b ? (a = a % b) : (b = b % a);
+}
+
+alert(`GCD of ${numA} and ${numB} is ${a + b} `);
+
+//Task 3
+dividers = [];
```
Comment (idovbush) [inline ba/js/lesson3/script.js]: wrong dividers is not initialized

File: ba/js/lesson3/script.js [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+
+let positive = 0;
+let negative = 0;
+let zero = 0;
+let odd = 0;
+let even = 0;
+console.log(positive, negative);
+for (let i = 0; i < 10; i++) {
+  let num = prompt("Enter a number....");
+  num = +num;
```
Comment (idovbush) [inline ba/js/lesson3/script.js]: this code is redundant, you can join to 50 code line

File: ba/js/lesson3/script.js [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+        `${firstNum} ${operationSign} ${secondNum} = ${firstNum * secondNum}`
+      );
+      break;
+    case "/":
+      console.log(
+        `${firstNum} ${operationSign} ${secondNum} = ${firstNum / secondNum}`
+      );
+      break;
+    default:
+      break;
```
Comment (idovbush) [inline ba/js/lesson3/script.js]: we do not need break statement in default section

File: ba/js/lesson3/script.js [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+  }
+  var continueCalc = prompt(
```
Comment (idovbush) [inline ba/js/lesson3/script.js]: do not use var

File: ba/js/lesson3/script.js [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+//Task 7
+let numberToShift = prompt("Enter a number  ");
+let shift = prompt("Enter a number of digits to shift  ");
+
+console.log(numberToShift.slice(shift) + numberToShift.slice(0, shift));
+
+//Task 8
+
+var today = 0;
+var day$;
```
Comment (idovbush) [inline ba/js/lesson3/script.js]: I m not sure about such naming day$

File: ba/js/lesson3/script.js (hunk 8 above)
Comment (OleksiiTarasenko) [reply ba/js/lesson3/script.js]: I had to use a global variable otherwise, the variable  "continueCalc" is not initialized in the loop condition.

**Pull request merged**